### ISSUE TITLE
New FMD Release (@ALL: READ THIS FIRST)

### ISSUE THREAD

**Issue by SDXC** [riderkick/FMD#1723]
Hi everyone

As FMD seems to be a little bit dead for some time now, I decided to take over the project by forking it and publishing the first update release on my repository.

You can find the new releases here (including install instructions):
https://github.com/fmd-project-team/FMD/releases

Please open new issues here:
https://github.com/fmd-project-team/FMD/issues

Find some help here before opening new issues:
https://github.com/fmd-project-team/FMD/wiki

Follow us on Discord:
https://discord.gg/cXKKgw3

**Comment by Ianunaut** [comment]
Works like a dream, my good man. God bless.

**Comment by Twoearle** [comment]
Based

**Comment by panhartstuff** [comment]
Nice.
But why do these FMD projects keep on getting abandoned...

**Comment by ChocolateOtaku** [comment]
Yeah, i hope this is not just another hype that will drop again after a few month when the new developers recognize how much of their free time and effort they have to sacrifice to keep up with all the existing problems and upcoming issues :(

I will wait a few month and watching the new project's activity carefully before i decide to switch back to FMD...

**Comment by SDXC** [comment]
I try to keep the project as open as possible. Meaning an open communication with the users, visible tracking of the issues, letting other users contribute to the project and so on.

Of course there will be things I can't handle alone skillwise, but if that happens, I will ask the community for help. Regarding the time and effort: I'll try to handle it a bit more leisurely. Keeping the priorities in check can help.

**Comment by kuso-ge** [comment]
@SDXC 
Hi, just a question regarding the recent issue with MangaDex retrieving 0 on current chapters.
You recon there's no work around other than just waiting it out?

**Comment by SDXC** [comment]
@bloodsign The Cloudflare workaround still works. It's just my personal opinion not to create more traffic for MD while it's already being targeted by DDoS. For most of the other sites I don't really care, because they monetize for the worse quality, etc. but in case of MD I want to be fair.

**Comment by kuso-ge** [comment]
> @bloodsign The Cloudflare workaround still works. It's just my personal opinion not to create more traffic for MD while it's already being targeted by DDoS. For most of the other sites I don't really care, because they monetize for the worse quality, etc. but in case of MD I want to be fair.

Yeah I just tried the cloudflare workaround from the FAQ and it did worked. 
I do read the series I follow on mangadex w/ ad-block disabled just to support the site, I just download a copy for re-reading or when I have to visit my farm(there's no internet/reception there)
Thanks btw!

**Comment by SDXC** [comment]
@bloodsign MD has no ads, so you can use adblock however you want. MD runs mostly on donations (I'm also a donor).
